public/src/components/channelManagement/campaigns/StickyCampaignBar.tsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { Theme, Typography, makeStyles } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+
+const useStyles = makeStyles(({ palette, spacing }: Theme) => ({
+  container: {
+    display: 'flex',
+    alignItems: 'flex-start',
+    justifyContent: 'space-between',
+    paddingLeft: spacing(3),
+    paddingRight: spacing(3),
+    paddingBottom: spacing(2),
+    paddingTop: spacing(1),
+    backgroundColor: palette.grey[200],
+    borderBottom: `1px solid ${palette.grey[500]}`,
+  },
+  namesContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  mainHeader: {
+    fontSize: '32px',
+    fontWeight: 'normal',
+  },
+  secondaryHeader: {
+    fontSize: '14px',
+    color: palette.grey[700],
+  },
+  lockContainer: {
+    alignSelf: 'flex-end',
+    display: 'flex',
+    '& > * + *': {
+      marginLeft: spacing(2),
+    },
+    marginLeft: spacing(1),
+  },
+  buttonText: {
+    fontSize: '14px',
+    fontWeight: 500,
+    textTransform: 'uppercase',
+    letterSpacing: '1px',
+    color: palette.grey[800],
+  },
+  icon: {
+    color: grey[700],
+  },
+}));
+
+interface StickyTopBarProps {
+  name: string;
+  nickname?: string;
+  existingNames: string[];
+  existingNicknames: string[];
+}
+
+const StickyTopBar: React.FC<StickyTopBarProps> = ({
+  name,
+  nickname,
+  existingNames,
+  existingNicknames,
+}: StickyTopBarProps) => {
+  const classes = useStyles();
+  const mainHeader = nickname ? nickname : name;
+  const secondaryHeader = nickname ? name : null;
+
+  // Purely to stop linting errors during development
+  console.log(existingNames, existingNicknames);
+
+  return (
+    <header className={classes.container}>
+      <div className={classes.namesContainer}>
+        <Typography variant="h2" className={classes.mainHeader}>
+          {mainHeader}
+        </Typography>
+        <Typography className={classes.secondaryHeader}>{secondaryHeader}</Typography>
+      </div>
+      {/*      
```

Comment:
See comments in other files. For now, I'm suppressing the display of `copy`, `edit` (and `archive`) buttons until after we've made decisions on how to handle these functionalities

Reply:
Removed code - we'll add in editing functionality in a future PR